applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py

```diff
@@ -239,6 +239,9 @@ class Simple2D2NBeamCrTest(StructuralMechanicsTestFactory):
 class InitialStateElasticityTest(StructuralMechanicsTestFactory):
     file_name = "InitialStateElasticity/initial_state_test"
 
+class InitialStrainShellQ4ThickTest(StructuralMechanicsTestFactory):
```

Comment:
I think the test is still missing in test_StructuralMechanicsApplication.py (maybe is added automatically, last I checked it wasn't)